convert_rmd.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import os
+from sanitize_html import sanitize
+from tempfile import NamedTemporaryFile
+from rpy2.robjects.packages import importr
+
+
+def convert(stream):
+    binary_data = stream.read()
+    raw_rmd = binary_data.decode('ascii')
+
+    safe_rmd = __sanitize_rmd(raw_rmd)
+    raw_html = __to_html(safe_rmd)
+    safe_html = sanitize(raw_html)
+
+    return safe_html
+
+
+def __sanitize_rmd(data: str) -> bytes:
```

Comment:
So this is not something that nh3 provides for rmd file, but they do for python notebooks? It might be worth it to explain why this is needed here, but not in the file above

Reply:
I'm not sure I understand the ask here; this is the step which strips code blocks from rmd before we process into HTML with knitr. nh3 is used afterwards to clean the HTML; HTML code injection attacks are a separate vulnerability.

Reply:
I'll add more comments, I think this wasn't clear enough in code!